neo/conf/application.conf

```diff
@@ -76,6 +76,7 @@ bounded-mailbox {
 }
 
 neo.timeout.initial = "10 minutes"
+neo.stream.max.frame.length = "8388608 Bytes"
```

Comment:
I fixed the code.
For the test case of Neo, can we compare the json result received in NeoActor to the json result sent by middleware, to test if we get all data right?